src/App.vue

```diff
@@ -44,15 +47,14 @@ export default {
       const user = confirmLogin(userData, this.allUsers)
       if (user) {
         this.updateLogin()
-        this.currentUser = user.id
+        setUserId(user.id)
```

Comment:
Good idea to start utilizing local storage instead of passing down the id through props all the time.